tests/xharness/Microsoft.DotNet.XHarness.iOS.Shared/Hardware/Simulators.cs

```diff
@@ -198,7 +195,7 @@ async Task<bool> CreateDevicePair (ILog log, ISimulatorDevice device, ISimulator
 			log.WriteLine ($"Creating device pair for '{device.Name}' and '{companion_device.Name}'");
 
 			var capturedLog = new StringBuilder ();
-			var pairLog = new CallbackLog ((string value) => {
+			var pairLog = new CallbackLog ((value) => {
```

Comment:
Looks like we have a number of unrelated changes (not moving, but others, spaces etc..) was that VS taking over?

Reply:
Yeah, looks like when you rename namespace, it auto-formats the file and all files where it is referenced.. :-/